packages/babel-plugin-proposal-class-properties/test/fixtures/private/tagged-template-static/exec.js

```diff
@@ -0,0 +1,17 @@
+class Foo {
+    static #tag() {
+      return this;
+    }
+    
+    static #tag2 = () => this
```

Comment:
Here we are not testing the receiver, since arrow functions always get `this` from the definition context and not depending on how they are called. We should use this instead:
```suggestion
    static #tag2 = function () { return this };
```

Reply:
Ops miss that one